plugins/woocommerce/tests/api-core-tests/tests/data/data-crud.test.js

```diff
@@ -1527,19 +1534,19 @@ test.describe( 'Data API tests', () => {
 						},
 						{
 							code: 'PS',
-							name: 'Jordanian dinar',
```

Comment:
I'm not an expert on the politics that might lie behind this, but this seems ... notable? Do we have somehow who can explain the background here before merging?

Reply:
Thanks for asking this question, this was also confusing for me too. 

I searched it online, and read a few websites telling about the currencies used in Palestine, and it seems Palestine doesn't have it's own currency, and there are three major currencies used, the US Dollar, the Israeli Shekel, and the Jordanian Dinar for trade, which the websites tell in common. But some websites talk about a legal tender used in West Bank and the Gaza Strip, and it changed from JOD to ILS after the Paris Protocol in 1994. 

https://www.monito.com/en/what-is-the-currency-in/palestine
https://israelrail.com/israeli-currency-new-israeli-shekel/
https://gopalestine.org/about-palestine__trashed/currency-and-money-in-palestine/
https://www.worldtravelguide.net/guides/middle-east/palestinian-national-authority/money-duty-free/
https://qr.ae/pyLUIP
https://globalfinancialdata.com/palestine

But they do not actually tell whether Palestine uses a definitive currency. They mostly talk about Israeli occupied territories using ILS as a legal tender.

And according to CLDR, there are currently two currencies used (ILS and JOD): 

https://github.com/tpaksu/cldr/blob/main/cldr/cldr-core/supplemental/currencyData.json#L2617-L2640

Besides those, I don't know exactly what's true. Maybe we can ask some A11ns who are living in that area?

Reply:
I discussed this with a few colleagues and thought about it more. In, for example, `i18n/countries.php`, we explicitly call out following the Unicode CLDR recommendation. They're the experts on these issues and provide the recommendations that we've decided to follow. Let's go forward with this one.

Reply:
Yes, but the CLDR doesn't explicitly call out any currencies on the four countries I've mentioned in the last linked review comment of yours. Maybe we should develop a feature that allows selecting the right currency for those countries which have inseperable currencies on onboarding?